spec/features/user_adds_new_data_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+# As an authenticated user
+# I want to contribute
+
+# Acceptance Criteria:
+# [ ] There is a add data button
+# [ ] I can succesfully add new data
+
+feature "sees profile in top bar" do
+  let!(:user) { User.create(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.